troncli/cli.py

```diff
@@ -13,78 +13,72 @@
 
     init_handler = h_init.Init()
     utils.progress_msg('Creating folders')
     init_handler.create_dirs()
     utils.progress_msg('Downloading release builds')
-    asyncio.run(init_handler.fetch_jars(version))
-    asyncio.run(init_handler.move_jars())
+
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(init_handler.fetch_jars(version))
+    loop.run_until_complete(init_handler.move_jars())
+    loop.close()
 
 
 @cbox.cmd
-def config(nettype: str, fullhttpport: int, solhttpport: int, fullgrpcport: int, solgrpcport: int):
-    """Create customize config files.
-    >> 
-    Settings: 
-        --nettype
-        --fullhttpport
-        --solhttpport
-        --fullgrpcport
-        --solgrpcport
-    """
-    # >>
-    # Example:
-    #     tron-cli config --nettype 'private' --fullhttpport 8500 --solhttpport 8600 --fullgrpcport 50051 --solgrpcport 5001
+def config(nettype: str,
+           fullhttpport: int,
+           solhttpport: int,
+           fullgrpcport: int,
+           solgrpcport: int):
+    """Create customize config files."""
 
     config_handler = h_config.Config()
     utils.progress_msg('Setting up config files')
-    asyncio.run(config_handler.init())
-    asyncio.run(config_handler.set_net_type(nettype))
-    asyncio.run(config_handler.set_http_port(fullhttpport, 'full'))
-    asyncio.run(config_handler.set_http_port(solhttpport, 'sol'))
-    asyncio.run(config_handler.set_grpc_port(fullgrpcport, 'full'))
-    asyncio.run(config_handler.set_grpc_port(solgrpcport, 'sol'))
-    asyncio.run(config_handler.export())
+
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(config_handler.init())
+    loop.run_until_complete(config_handler.set_net_type(nettype))
+    loop.run_until_complete(config_handler.set_http_port(fullhttpport, 'full'))
+    loop.run_until_complete(config_handler.set_http_port(solhttpport, 'sol'))
+    loop.run_until_complete(config_handler.set_grpc_port(fullgrpcport, 'full'))
+    loop.run_until_complete(config_handler.set_grpc_port(solgrpcport, 'sol'))
+    loop.run_until_complete(config_handler.export())
+    loop.close()
 
 
 @cbox.cmd
 def run(nodetype: str):
-    """Run node.
-    >> 
-    Settings: 
-        --nodetype
-    """
+    """Run node."""
     utils.progress_msg('Starting node(s)')
     worker = h_worker.Worker()
-    asyncio.run(worker.run(nodetype))
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(worker.run(nodetype))
+    loop.close()
 
 
 @cbox.cmd
 def stop(pid: str):
-    """Stop node.
-    >> 
-    Settings: 
-        --pid
-    """
+    """Stop node."""
     worker = h_worker.Worker()
     utils.progress_msg('Shutting down node(s)')
-    asyncio.run(worker.stop(pid))
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(worker.stop(pid))
+    loop.close()
 
 
 @cbox.cmd
 def quick():
     """Quick start. (run a full private node by one command)
-    >>
-    Example:
-        tron-cli quick
+
```

Comment:
this will be output as help information by cbox in the command line, let's try to make it consistent